packages/swingset-liveslots/tools/fakeVirtualSupport.js

```diff
@@ -158,6 +159,12 @@ export function makeFakeLiveSlotsStuff(options = {}) {
     return exportID;
   }
 
+  function allocatePromiseID() {
+    const exportID = nextExportID;
+    nextExportID += 1;
+    return exportID;
+  }
```

Comment:
It would feel more consistent to use a separate ID
```suggestion
  let nextPromiseID = 1;
  function allocatePromiseID() {
    const promiseID = nextPromiseID;
    nextPromiseID += 1;
    return promiseID;
  }
```